Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,51 @@
+@import 'colors';
+
+%card-radius {
+  -webkit-border-radius: 5px;
+  -moz-border-radius: 5px;
+  border-radius: 5px;
+}
+
+.tech {
+  height: 464px;
+  margin: auto;
+  padding: 16px;
+  width: 320px;
+
+  .base{
+    @extend %card-radius;
+    background: white;
+    border: 2px $green solid;
+    height: 100%;
+
+    .upper {
+      @extend %card-radius;
+      align-items: center;
+      background: $green;
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+      margin: 16px 0 0 16px;
+      padding: 16px;
+      width: 100%;
+
+      img {
+        height: 144px;
+        margin-bottom: 16px;
+      }
+
+      h4, span {
+        font-size: 20.8px;
```

Comment:
Mmm why decimals? Also try not to set styles to tags. Add them to a class

Reply:
I did development in rems and when I did conversion, it returns that value. I am going to put 21px indeed 20.8px